shared/editor/embeds/index.tsx

```diff
@@ -55,6 +55,8 @@ export class EmbedDescriptor {
   defaultHidden?: boolean;
   /** Whether the bottom toolbar should be hidden – use this when the embed itself includes a footer */
   hideToolbar?: boolean;
+  /** Whether the embed should be auto matched when pasting a URL */
+  noAutoMatch?: boolean;
```

Comment:
```suggestion
  matchOnPaste?: boolean;
```

Maybe we just get explicit with this?